docs/articles/nunit/writing-tests/attributes/explicit.md

```diff
@@ -38,9 +38,32 @@ However, the following options will **not** include explicit tests
     --where cat!=X
 ```
 
+## TRAP
```

Comment:
@OsirisTerje is it your intention to provide a warning here? If so I can update the formatting accordingly.